packages/curve-ui-kit/src/features/user-profile/home/Home.tsx

```diff
@@ -0,0 +1,25 @@
+import Stack from '@mui/material/Stack'
+
+import { useConnectWallet, getWalletSignerAddress } from '@ui-kit/features/connect-wallet'
+import { SizesAndSpaces } from '@ui-kit/themes/design/1_sizes_spaces'
+
+import { UserProfileHeader } from './Header'
+import { Settings } from '../settings'
+
+const { Spacing } = SizesAndSpaces
+
+type Props = {
+  onClose: () => void
+}
+
+export const Home = ({ onClose }: Props) => {
```

Comment:
This component's name isn't super clear

Reply:
It derives from the multiple pages that are planned once the user profile is fully implemented. But it's subject to change.